src/routes/MainRoute.js

```diff
@@ -32,6 +34,8 @@ const MainRoute = () => {
         <Route path="/createPostModal_3" component={CreatePostModal_3} />
         <Route path="/addActivity" component={AddActivityModal} />
         <Route exact path="/profilePage_test" component={ProfilePage_example} />
+        <Route path="/choosingUser" component={ChoosingUser} />
+        <Route path="/login" component={LoginPage} />
```

Comment:
So with this one, can u talk again what screen will be shown first, Home Screen or Authentication Screen?
Becuz, some situations of other products, they let's people see the Home Screen and then if they want to choose the project, they will be navigated to Authentication Screen.

Reply:
**Yes. Our homescreen will be the View Projects screen**
I switch the positions of the routes for those 2 routes to be the first for the sake of testing and development
When rolling out, as we've discussed before, the home screen is View Projects Screen
@TrNgTien